assets/resourceSnippets/Automanage Best Practices DevTest Assignment.snippet.json

```diff
@@ -7,9 +7,10 @@
     },
     "resources": [
         {
-            "type": "Microsoft.Compute/virtualMachines/providers/configurationProfileAssignments",
+            "type": "Microsoft.Automanage/configurationProfileAssignments",
+            "scope": "'Microsoft.Compute/virtualMachines/${1:vmName}",
```

Comment:
Can you move the `scope` property above the `type` property - a little more readable and the common patther.

Reply:
Done